tutorials/workflow/java/task-chaining/src/test/java/io/dapr/springboot/examples/TaskChainingAppTests.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2025 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.springboot.examples;
+
+import io.dapr.client.DaprClient;
+import io.dapr.springboot.DaprAutoConfiguration;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+import static io.dapr.springboot.examples.StringMatchesUUIDPattern.matchesThePatternOfAUUID;
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest(classes = {TestTaskChainingApplication.class, DaprTestContainersConfig.class,
+        DaprAutoConfiguration.class, },
+        webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+class TaskChainingAppTests {
+
+  @Autowired
+  private DaprClient daprClient;
+
+  @BeforeEach
+  void setUp() {
+    RestAssured.baseURI = "http://localhost:" + 8080;
+    org.testcontainers.Testcontainers.exposeHostPorts(8080);
+  }
+
+
+  @Test
```

Comment:
I love that tests are added, thank you! That's also something to include for the .NET and Python tutorials.